src/stacked_borrows.rs

```diff
@@ -164,18 +165,19 @@ pub struct Stacks {
 /// Core per-location operations: deref, access, create.
 /// We need to make at least the following things true:
 ///
-/// U1: After creating a Uniq, it is at the top (+unfrozen).
-/// U2: If the top is Uniq (+unfrozen), accesses must be through that Uniq or pop it.
-/// U3: If an access (deref sufficient?) happens with a Uniq, it requires the Uniq to be in the stack.
+/// U1: After creating a `Uniq`, it is at the top (and unfrozen).
+/// U2: If the top is `Uniq` (and unfrozen), accesses must be through that `Uniq` or pop it.
+/// U3: If an access (deref sufficient?) happens with a `Uniq`, it requires the `Uniq` to be in the stack.
 ///
-/// F1: After creating a &, the parts outside `UnsafeCell` are frozen.
+/// F1: After creating a `&`, the parts outside `UnsafeCell` are frozen.
 /// F2: If a write access happens, it unfreezes.
-/// F3: If an access (well, a deref) happens with an & outside `UnsafeCell`, it requires the location to still be frozen.
+/// F3: If an access (well, a deref) happens with an & outside `UnsafeCell`, it requires the location to still be
+///     frozen.
```

Comment:
just one left. maybe wrap this one at the comma?

Reply:
Sure.